CBLClient/Apps/CBLTestServer-Java-Desktop/build.gradle

```diff
@@ -0,0 +1,66 @@
+plugins {
+    id 'java'
+}
+
+group 'com.couchbase.mobiletestkit'
+
+ext {
+    DEF_VERSION = "2.7.0-56"
+    COUCHBASE_LITE_JAVA_VERSION = System.getProperty('version', DEF_VERSION)
+
+    PROJECT_DIR = "${projectDir}"
+    ROOT_DIR = "${PROJECT_DIR}/.."
+    JAVA_COMMON_DIR = "${ROOT_DIR}/JavaRequestHandlers"
+    ASSET_DIR = "${JAVA_COMMON_DIR}/src/main/assets"
+    JAVA_SERVER_DIR = "${ROOT_DIR}/JavaRequestListener"
+}
+
+
+sourceCompatibility = 1.8
+
+jar {
+    archiveBaseName = "CBLTestServer-Java"
+    archiveAppendix = "Desktop"
+    archiveVersion = "${COUCHBASE_LITE_JAVA_VERSION}"
```

Comment:
you can just write:
archiveVersion = COUCHBASE_LITE_JAVA_VERSION

Reply:
you are right :) since archiveName is deprecated, I was following the documentation to fill up all variables, which all have static values, really don't need to. Will fix it

Reply:
fixed